rhp/v4/server.go

```diff
@@ -0,0 +1,1078 @@
+package rhp
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"time"
+
+	"go.sia.tech/core/consensus"
+	rhp4 "go.sia.tech/core/rhp/v4"
+	"go.sia.tech/core/types"
+	"go.sia.tech/coreutils/chain"
+	"go.sia.tech/coreutils/wallet"
+	"go.uber.org/zap"
+	"lukechampine.com/frand"
+)
+
+// maxBasisDiff is the maximum number of blocks by which a transaction's basis
+// can differ from the current tip.
+const maxBasisDiff = 20
+
+var protocolVersion = [3]byte{0, 0, 1}
```

Comment:
...isn't this version 4? 😑

Reply:
I will note that was a left over from the summit